harmony/plugin.yaml

```diff
@@ -2,24 +2,39 @@
 plugin:
     # Global plugin attributes
     type: interface                 # plugin type (gateway, interface, protocol, system, web)
+
     description:
-        de: 'Harmony Hub plugin, **seit SmartHomeNG v1.3**'
-        en: ''
+        de: 'Harmony Hub plugin **seit SmartHomeNG v1.3**'
+        en: 'Harmony hub plugin'
+
     maintainer: pfischi
-#    tester:                        # Who tests this plugin?
-#    keywords: iot xyz
-#    documentation: https://github.com/smarthomeNG/smarthome/wiki/CLI-Plugin        # url of documentation (wiki) page
+    documentation: https://github.com/smarthomeNG/plugins/blob/develop/harmony/README.md
     support: https://knx-user-forum.de/forum/supportforen/smarthome-py/1046500-harmony-hub-plugin
 
-    version: 1.3.0.5               # Plugin version
+    version: 1.4.0                 # Plugin version
     sh_minversion: 1.3             # minimum shNG version to use this plugin
-#    sh_maxversion:                 # maximum shNG version to use this plugin (leave empty if latest)
     multi_instance: False          # plugin supports multi instance
     classname: Harmony             # class containing the plugin
 
 parameters:
-    # Definition of parameters to be configured in etc/plugin.yaml
-    
+  harmony_ip:
+    type: int
+    description:
+      de: "Harmony Hub IP-Adresse"
+      en: "Harmony hub ip address"
+
+  harmony_port:
+    type: int
+    description:
+      de: "(optional) Harmony Hub Port. Default: 5222"
+      en: "(optional) Harmony hub port. Default: 5222"
+
+  sleekxmpp_debug:
+    type: bool
+    description:
+      de: "(optional) Debug-Ausgaben der sleekxmpp-Klasse. Default: false"
+      en: "(optional) Debug print of ht esleekxmpp class. Default: false"
```

Comment:
The "print of ht eslee" should be "print of the slee".

Reply:
done